tests/data/utils.py

```diff
@@ -24,7 +24,9 @@ def iterate_over_rules():
         leaf_dir = os.path.basename(dir_name)
         rel_dir = '.' + dir_name[len(_DIR):]
         if leaf_dir.startswith('rule_'):
-            result = Rule(rel_dir, _SSG_PREFIX + leaf_dir, files)
+            # use only .sh files
```

Comment:
This comment doesn't add much to the line that comes after it. I would suggest to explain why we only care about .sh files and nothing else. What are we skipping? What do the .sh files contain?